modules/troubleshooting-cluster-upgrader.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * upgrade/upgrade-roxctl.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="troubleshooting_upgrader_{context}"]
+= Troubleshooting Cluster Upgrader
```

Comment:
```suggestion
= Troubleshooting the cluster upgrader
```
Typically we don't capitalize pieces or components of the software (with a few exceptions, like "Central, Sensor") so I think not capitalized here is the way to go.